food_ratings/frontend/views.py

```diff
@@ -56,7 +56,6 @@ def _entry(register, key):
     response = _get('%s/record/%s.json' % (url, key))
     return response.json()
 
-
```

Comment:
Ending life of that worthless empty line wasn't worthy drawing your attention. As it's ironically happened let me express my highest disappointment as well as respect for the line that it has remained over there for so long. To put it this way: **NOT ON MY WATCH!**